Frameworks/CoreText/CTLine.mm

```diff
@@ -38,7 +44,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     ret->_ascent = _ascent;
     ret->_descent = _descent;
     ret->_leading = _leading;
-    ret->_runs = [_runs copy];
```

Comment:
`-[NSArray copy]` absolutely returns a new array with _retained_ references to the items inside it. So long as either array is around, the objects inside them will live on.